src/typedefs.jl

```diff
@@ -13,11 +13,11 @@
 #
 # TODO: Ensure that size(values) == size(isnull) using inner constructor.
 # TODO: Implement outer constructor required once we add an inner constructor.
-@doc """
-`NullableArray{T, N}` is an efficient alternative to `Array{Nullable{T}, N}`.
+"""
+    NullableArray{T, N} is an efficient alternative to `Array{Nullable{T}, N}`.
```

Comment:
This line isn't a signature, it's just a sentence, so keep it as it was previously.